include/swift/AST/DiagnosticsParse.def

```diff
@@ -1321,6 +1321,8 @@ ERROR(attr_renamed, none,
       "'@%0' has been renamed to '@%1'", (StringRef, StringRef))
 WARNING(attr_renamed_warning, none,
         "'@%0' has been renamed to '@%1'", (StringRef, StringRef))
+ERROR(attr_name_close_match, none,
+      "No attribute named '@%0', did you mean '@%1'?", (StringRef, StringRef))
```

Comment:
Sorry for the late comment, but this should be lowercase! And to be nitpicky, the comma should be a semicolon.

Reply:
My bad! I'll submit a new PR with those changes tonight.